contributing.md

```diff
@@ -15,8 +15,11 @@ You'll need a [GitHub account](https://github.com/join)!
 1. Go edit the page containing the list of supported plugins by clicking [this link](https://github.com/jfmengels/eslint-rule-documentation/edit/master/plugins.json).
 2. Find out where the documentation for the rules of the plugin are located. Find the plugin on `npm`, then the location of the repository, then try to find out the generic url for rule documentation. Very often, it is at `https://github.com/USERNAME/eslint-plugin-PLUGINNAME/blob/master/docs/rules/RULENAME.md`.
 3. Add a new entry for your plugin, using the link in step 2, containing `RULENAME`as a placeholder for the name of the rule.
-Example: For `eslint-plugin-foobar`, whose documentation files are Markdown files in `https://github.com/theauthor/eslint-plugin-foobar/blob/master/docs/rules/`, add a new entry of the form:
-  - `"foobar": "https://github.com/theauthor/eslint-plugin-foobar/blob/master/docs/rules/RULENAME.md"`
+   - If the author uses the standard convention of hosting the documentation at <code>https://github.com/*author*/eslint-plugin-*plugin-name*/blob/master/docs/rules/*rule-name*.md</code>, then you can just put <code>"*plugin-name*": "*author*",</code> in `plugins.json`.
+   - If the author did not follow the repository naming convention outlined above, but still puts the rule documentation in <code>docs/rules/*rule-name*.md</code>, then you can put <code>"*plugin-name*": "*author*/*repo-name*",</code> in `plugins.json`.
```

Comment:
Although the `<code>` blocks aren't broken here, they are a different style than the rest, just use ` to match the rest of the document.